server/webapp/WEB-INF/rails.new/app/views/navigation_elements/_navigation.html.erb

```diff
@@ -1,41 +1,57 @@
 <nav class="top-bar navigation" data-topbar role="navigation">
   <ul class="title-area">
     <li class="name">
-      <a href="#" class="logo"></a>
-    </li>
-    <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
+      <%= link_to '', main_app.pipeline_dashboard_path, class: "logo" -%>    </li>
     <li class="toggle-topbar menu-icon"><a href="#"><span>Menu</span></a></li>
   </ul>
 
   <section class="top-bar-section">
-    <ul class="right">
-        <li class="has-dropdown">
-            <a href="#" class="user">admin</a>
+    <% unless @no_tabs %>
```

Comment:
AFAIK — this new set of view partials is only used on the new pipeline config edit SPA, no other controller(s) use it at this time.

You seem to have copied it from existing view partials, that seem to be used in a lot more than one place, so there's some view logic in there to deal with that.